src/dosext/mfs/lfn.c

```diff
@@ -726,6 +726,8 @@ static int wildcard_delete(char *fpath, int drive)
 		free(dir);
 		return lfn_error(PATH_NOT_FOUND);
 	}
+
+	// coverity [bad_alloc_strlen:FALSE]
```

Comment:
What was the problem?

Reply:
Coverity thinks `malloc(strlen(p + 1))` should be `malloc(strlen(p) + 1)` and is indicative of a typo. In this case the `p + 1` is the significant part of our string and so is correct. I did think about changing it to `malloc(strlen(&p[1]))` but was conscious of not making the code less readable just to please Coverity.

Reply:
Interesting, as there is already a
`+1` after `)` too! It had to notice this,
so might be a bug in coverity.